### DISCUSSION TITLE
OPNSense dashboard

### DISCUSSION THREAD

**Discussion by DannyJanmaat** [bsmithio/OPNsense-Dashboard#67] [Show and tell]
.

**Comment by tismofied** [comment]
Please do.

**Comment by tismofied** [comment]
> I made a renewed version of the OPNSense-dashboard partially based on the one bsmithio made.
> 
> 
> 
> My version is based on the latest versions : 
> 
> Ubuntu 22.04.4 LTS (Feb 22, 2024)
> 
> OPNSense 24.1.7 (May 16, 2024)
> 
> OPNSense plugin, Telegraf 1.12.10 (Apr 26, 2024)
> 
> InfluxDB 2.7.7 (Apr 12, 2024)
> 
> Graylog 5.2.7 (Apr 30, 2024)
> 
> Graylog distribution, OpenSearch 2.14.0 (May 14, 2024)
> 
> OpenSearch, Lucene 9.10.0 (Feb 20, 2024)
> 
> Grafana Enterprise 11.0.0 (May 14, 2024)
> 
> MongoDB 7.0.9 (Apr 29, 2024)
> 
> Zabbix 6.4.0 (March 6, 2023), also works with BETA-version 7.0.0beta2.
> 
> 
> 
> I have all documented and will later release an instruction :).
> 
> Maybe if I have time I make an instruction on how to do it via Docker ;).

Whatever happened to this?

**Reply by DannyJanmaat** [reply]
No time, but I'm planning to do it in my vacation over a couple of weeks.

**Reply by tismofied** [reply]
Thank you and looking forward to your project
